src/components/DevTools/data/cTypes.json

```diff
@@ -34,12 +35,13 @@
       }
     },
     "owner": "ROOT_ATTESTER",
-    "hash": "0xde9f624875aa620d06434603787a40c8cd02cc25c7b775cf50de8a3a96bbeafa"
+    "hash": "0x5366521b1cf4497cfe5f17663a7387a87bb8f2c4295d7c40f3140e7ee6afc41b"
   },
   "IS_OFFICIAL": {
     "schema": {
       "$id": "IsOfficial",
```

Comment:
should be - `kilt:ctype: 0x5b23725ed650571b80b11ca6fc2d017a5716e9dc70c618b973c2994ae0eeb0a3`